programming/editors/vscode.md

```diff
@@ -35,3 +35,57 @@ you'll need to make the library available within the Python environment that you
 3. Run `pip install sr.robot3`.
 
 [code-completion]: https://en.wikipedia.org/wiki/Autocomplete#In_source_code_editors
+
+## Remote Debugging
+
+<div class="info">
+This documentation refers to a feature which is only available from software version `2023.1.0` and later.
+</div>
+
+When connected to the [robot's WiFi hotspot](/docs/kit/wifi), it is possible to attach VS Code's
+debugger to the robot by performing the following steps:
+
+1. Ensure the [Python extension](#python-extension) is installed
```

Comment:
nit (also below): it looks like the prevailing style is to treat each bullet in the list as a sentence and thus include a full stop after each (if there isn't other punctuation).

Reply:
277dbb8